### ISSUE TITLE
baremetal build broke because of added support for RHEL8

### ISSUE THREAD

**Issue by bengland2** [openshift-metal3/dev-scripts#644]
**Describe the bug**

"make" fails to deploy a baremetal cluster in an ansible playbook within metal3-dev-env/vm-setup/roles/libvirt, apparently because of how it conditionalizes tasks for RHEL7 vs RHEL8.
This did not happen when I cloned on June 24, but it happened today.

**To Reproduce**

```
# for metal3/dev-scripts
export KUBECONFIG=~/dev-scripts/ocp/auth/kubeconfig
export ADDN_DNS=navi.alias.bos.scalelab.redhat.com
export EXT_IF=em3
export PRO_IF=em1
export INT_IF=p7p1
export WORKING_DIR=/root/dev-scripts
export NODES_PLATFORM=baremetal
export NODES_FILES=$WORKING_DIR/ironic_nodes.json
export MASTER_NODES_FILE=$WORKING_DIR/nodes.json
git clone https://github.com/openshift-metal3/dev-scripts
cd dev-scripts
make
```

**Expected/observed behavior**

```
TASK [libvirt : Install packages for libvirt] ******************************************************************************************************************************
task path: /root/dev-scripts/metal3-dev-env/vm-setup/roles/libvirt/tasks/install_setup_tasks.yml:5
fatal: [localhost]: FAILED! => {
    "msg": "The conditional check 'ansible_os_family == \"RedHat\" and ansible_lsb.major_release|int == 7' failed. 
The error was: error while evaluating conditional (ansible_os_family == \"RedHat\" and ansible_lsb.major_release|int == 7): 'dict object' has no attribute 'major_release'\n\n
The error appears to be in '/root/dev-scripts/metal3-dev-env/vm-setup/roles/libvirt/tasks/install_setup_tasks.yml': line 5, column 3,
 but may be elsewhere in the file depending on the exact syntax problem.
The offending line appears to be:
# different list of packages.
- name: Install packages for libvirt
 ^ here"
}
```

**Comment by cybertron** [comment]
This will be fixed when #643 merges.

**Comment by russellb** [comment]
Merged
